webpush_channels/__init__.py

```diff
@@ -29,7 +29,7 @@
     'project_docs': 'https://webpush-channels-broadcasting.readthedocs.io/',
     'multiauth.authorization_policy': (
         'kinto.authorization.AuthorizationPolicy'),
-    'http_api_version': HTTP_API_VERSION
+    'http_api_version': HTTP_API_VERSION,
```

Comment:
It is not, we can remove it, I added some settings that I removed after and the comma stayed.